.github/workflows/problem-matchers/doxygen.json

```diff
@@ -10,7 +10,7 @@
       "owner": "doxygen",
       "pattern": [
         {
-          "regexp": "^\\/src\\/([^:]+):(\\d+): ?(\\w+): ?(.*)$",
+          "regexp": "^.*?\\/src\\/([^:]+):(\\d+): ?(\\w+): ?(.*)$",
```

Comment:
If the bit before the `/src/` is optional does it need to be grouped, i.e. `"^(.+)?\\/src\\/ ..."`?

Reply:
They're only grouped here so they can be referenced in the subsequent lines and fed back into the annotation. The two different line formats look like this:

```
Parsing file /src/python/openassetio/plug/src/doc/src/Thumbnails.dox:33: warning: unable to resolve reference to `openassetio.managerAPI.ManagerInterface.ManagerInterface.getEntityMetadata' for \ref command
/src/python/openassetio/constants.py:148: warning: unable to resolve reference to `DocumentLocale' for \ref command
```

We don't care about the `Parsing file...` bit.